BackofficeBundle/Form/Type/Component/ContentSearchType.php

```diff
@@ -4,29 +4,44 @@
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
+use OpenOrchestra\Transformer\LuceneToBddTransformer;
 
 /**
  * Class ContentSearchType
  */
 class ContentSearchType extends AbstractType
 {
+    protected $transformer;
+
+    /**
+     * @param string $transformerClass
+     */
+    public function __construct($transformerClass)
```

Comment:
Agree with you @itkg-alavieille. But as the purpose, here, is to store a mongo condition after final user write a lucene condition in order to make this work once in write context and not many times in read context, it seems to me to be the best way. But it could be documented and be more safe with a test on interface as specified by @itkg-ngilain .